azure-pipelines.yml

```diff
@@ -19,7 +19,7 @@ stages:
 
   - job: security_analysis
     displayName: Security Analysis
-
+    condition: and(always(), not(variables['IS_DEMO_PIPELINE']))
```

Comment:
What does always() mean here? can't you just have the not(...) without the and?

Reply:
fixed